spec/requests/api/v1/provider_spec.rb

```diff
@@ -275,6 +279,42 @@ def get_provider_codes_from_body(body)
       end
     end
 
+    context 'with multiple recruitment cycles' do
+      describe 'JSON body response' do
+        let(:provider) { create(:provider, recruitment_cycle: current_cycle) }
+        let(:current_cycle) { build(:recruitment_cycle, year: '2019') }
+        let(:provider2) { create(:provider, recruitment_cycle: next_cycle) }
+        let(:next_cycle) { build(:recruitment_cycle, year: '2020') }
+
+        before do
+          provider
+          provider2
+          get_index
+        end
+
+        context 'with no cycle specified in the route' do
+          let(:get_index) { get '/api/v1/providers', headers: { 'HTTP_AUTHORIZATION' => credentials } }
+
+          it 'defaults to the current cycle when year' do
+            returned_provider_codes = get_provider_codes_from_body(response.body)
+
+            expect(returned_provider_codes).not_to include provider2.provider_code
+            expect(returned_provider_codes).to include provider.provider_code
+          end
+        end
+        context 'with a future recruitment cycle specified in the route' do
+          let(:get_index) { get '/api/v1/2020/providers', headers: { 'HTTP_AUTHORIZATION' => credentials } }
```

Comment:
wondering should this be condense with 

for current
`'/api/v1/providers'`
`'/api/v1/#{current}/providers'`

for future
`'/api/v1/#{future}/providers'`

Reply:
Seems like it's longer than just having 2020 though right?